robot/basestation/templates/ArmMotorTable.html

```diff
@@ -0,0 +1,50 @@
+<h5 class="container-header">Motor Table</h5>
```

Comment:
Are new elements to the table probable to be added and removed frequently? I don't think it falls in the scope of this PR considering it doesn't add that much value. The code for generation may end up being more confusing then the table.
Although Jinja (the thing flask uses for templatification) has for loops built in. I would open an issue for this if you guys agree it would be useful.

Reply:
I agree with @MartensCedric , out of scope for this PR. And new elements are not probable at all to be added/removed frequently.